pkg/scheduler/api/job_info.go

```diff
@@ -142,9 +142,10 @@ func getJobID(pod *v1.Pod) JobID {
 	return ""
 }
 
-func getTaskID(pod *v1.Pod) TaskID {
```

Comment:
TaskID is easy to read and it stands for UID.  Do not suggest to change it.

Reply:
Yeah, TaskID stands for UID, but the content of this func does not return UID at all.